visualization/dashboard/src/MLIDashboard/Controls/CohortEditor/CohortEditor.styles.ts

```diff
@@ -1,214 +1,254 @@
-import { FontSizes, FontWeights, getTheme, IProcessedStyleSet, IStyle, mergeStyleSets } from "office-ui-fabric-react";
+import { FontSizes, FontWeights, getTheme, IProcessedStyleSet, IStyle, mergeStyleSets, ICalloutContentStyles } from "office-ui-fabric-react";
 
 export interface ICohortEditorStyles {
-  wrapper:IStyle;
-  leftHalf:IStyle;
-  rightHalf:IStyle;
-  detailedList:IStyle;
-  filterHeader:IStyle;
-  addFilterButton:IStyle;
-  featureTextDiv:IStyle;
-  featureComboBox:IStyle;
-  operationComboBox:IStyle;
-  valueSpinButton:IStyle;
-  valueSpinButtonDiv:IStyle;
-  minSpinBox:IStyle;
-  maxSpinBox:IStyle;
-  featureText:IStyle;
-  treatCategorical:IStyle;
-  defaultText:IStyle;
-  existingFilter: IStyle;
-  filterLabel: IStyle;
-  defaultFilterList: IStyle;
-  container: IStyle;
-  addedFilter:IStyle;
-  addedFilterDiv:IStyle;
-  filterIcon:IStyle;
-  cohortName:IStyle;
-  saveCohort:IStyle;
-  saveAndCancelDiv:IStyle;
-  saveFilterButton:IStyle;
-  cancelFilterButton:IStyle;
+    wrapper: IStyle;
+    leftHalf: IStyle;
+    rightHalf: IStyle;
+    detailedList: IStyle;
+    filterHeader: IStyle;
+    addFilterButton: IStyle;
+    featureTextDiv: IStyle;
+    featureComboBox: IStyle;
+    operationComboBox: IStyle;
+    valueSpinButton: IStyle;
+    valueSpinButtonDiv: IStyle;
+    minSpinBox: IStyle;
+    maxSpinBox: IStyle;
+    featureText: IStyle;
+    treatCategorical: IStyle;
+    defaultText: IStyle;
+    existingFilter: IStyle;
+    filterLabel: IStyle;
+    defaultFilterList: IStyle;
+    container: IStyle;
+    addedFilter: IStyle;
+    addedFilterDiv: IStyle;
+    filterIcon: IStyle;
+    cohortName: IStyle;
+    saveCohort: IStyle;
+    saveAndCancelDiv: IStyle;
+    saveFilterButton: IStyle;
+    cancelFilterButton: IStyle;
+    closeIcon: IStyle;
+    cohortEditor: IStyle;
 }
 
 export const cohortEditorStyles: () => IProcessedStyleSet<ICohortEditorStyles> = () => {
     const theme = getTheme();
     return mergeStyleSets<ICohortEditorStyles>({
+        cohortEditor: {
+            position: 'absolute',
+            overflowY: 'visible',
+            width: '560px',
+            height: '624px',
+            left: '500px',
+            top: '100px',
+            //TODO: fix rgba here
```

Comment:
yeah, I've got a few of these. We'll just do a find and replace when we have time